tests/e2e/test_easylink_run.py

```diff
@@ -86,15 +86,26 @@ def test_easylink_run(pipeline_specification, input_data, computing_environment,
 
         # Check that implementation configuration worked
         diagnostics_dir = results_dir / "diagnostics"
-        assert load_yaml(diagnostics_dir / "1_step_1" / "diagnostics.yaml")["increment"] == 1
         assert (
-            load_yaml(diagnostics_dir / "2_step_2" / "diagnostics.yaml")["increment"] == 100
+            load_yaml(diagnostics_dir / "step_1_python_pandas" / "diagnostics.yaml")[
```

Comment:
After talking with Zeb, it seems we prefer naming the results etc. based on the implementation names instead of the former step id function.